MetaLamp/lending-pool/package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
@performanceArtist is it okay to push that file, or better to ignore it?

Reply:
This file should be present in diff.

Reply:
Now I see, that file appeared from mistaken `npm install` call from `lending pool` directory, thanks

Reply:
removed